resample/jackknife.py

```diff
@@ -0,0 +1,185 @@
+"""
+Jackknife resampling.
+"""
+
+from typing import Callable, Sequence, Generator
+import numpy as np
+
+
+def resample(sample: Sequence, copy: bool = True) -> Generator[np.ndarray, None, None]:
+    """
+    Generator of jackknifed samples.
+
+    Parameters
+    ----------
+    sample: array-like
+        Sample. If the sequence is multi-dimensional, the first dimension must
+        walk over i.i.d. observations.
+    copy: bool, optional
+        If true, return the replicated sample as a copy, otherwise return a view into
```

Comment:
```suggestion
        If `True`, return the replicated sample as a copy, otherwise return a view into
```
Also consistent with `False` later in the docstring